src/components/LoadingOverlay/LoadingOverlay.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import {Loader} from '../Loader';
+import type {LoaderProps} from '../Loader';
+import {Box} from '../layout';
+import type {BoxProps} from '../layout';
+import {block} from '../utils/cn';
+
+import './LoadingOverlay.scss';
+
+const b = block('loading-overlay');
+
+export type OverlayBackground = 'base' | 'float';
+
+export interface LoadingOverlayProps extends BoxProps {
+    loaderProps?: LoaderProps;
+    background?: OverlayBackground;
+    loading?: boolean;
+    children?: React.ReactNode;
+    className?: string;
+    contentClassName?: string;
+}
+
+export function LoadingOverlay({
+    loaderProps,
```

Comment:
do we realy need override className in Loader component? 
I think it's not worth giving the user this opportunity to mix class name.
instead of `loaderProps` let pass `size` prop only?